sdgym/evaluate.py

```diff
@@ -0,0 +1,369 @@
+import json
+import logging
+
+import numpy as np
+import pandas as pd
+from pomegranate import BayesianNetwork
+from sklearn.ensemble import AdaBoostClassifier
+from sklearn.linear_model import LinearRegression, LogisticRegression
+from sklearn.metrics import accuracy_score, f1_score, r2_score
+from sklearn.mixture import GaussianMixture
+from sklearn.neural_network import MLPClassifier, MLPRegressor
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.tree import DecisionTreeClassifier
+
+from sdgym.constants import CATEGORICAL, CONTINUOUS, ORDINAL
+
+LOGGER = logging.getLogger(__name__)
+
+
+_MODELS = {
```

Comment:
In master branch, classifiers / regressors are tuned for each dataset. Please use the same settings as master branch.

Reply:
This is implicitly included in the `project_type` that we added to the metadata jsons.